src/Java/Lesson5/CivilAviation.java

```diff
@@ -0,0 +1,43 @@
+package Java.Lesson5;
+
+import java.util.Scanner;
+
+public class CivilAviation extends AirTransport {
+    private static int passengers;
+    int numberOfPassengers;
+    boolean businessClassAvailability;
```

Comment:
@Ihar-Malinouski лучше так
```suggestion
    boolean isBusinessClass;
```